### PR TITLE
Github actions

### PR DESCRIPTION
**Pull request by claireh93**
New workflows to deploy docs, run testing and publish coverage through github actions.

### CODE REVIEW

File: test/test_metrics.jl [hunk 1]
```diff
@@ -3,6 +3,10 @@ module TestMetrics
 using Phylo
 using Test
 
+if VERSION < v"1.5"
```
Comment (richardreeve) [inline test/test_metrics.jl:6]: Should add Compat to test's deps and `using Compat: contains` here.

File: docs/make.jl [hunk 2]
```diff
@@ -3,13 +3,11 @@ using Phylo
 
 if VERSION ≥ v"1.1.0"
```
Comment (richardreeve) [inline docs/make.jl:4]: Can drop this now I assume?

File: Project.toml [hunk 3]
```diff
@@ -16,24 +16,25 @@ Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
...
 Missings = "^0.4"
 RecipesBase = "0.6, 0.7, 0.8,1"
 Requires = "^1"
 SimpleTraits = "^0.9"
 Tokenize = "^0.5"
 Unitful = "^1"
+julia = "^1"
 
 [extras]
 JuliaDB = "a93385a2-3734-596a-9a66-3cfbb77141e6"
```
Comment (richardreeve) [inline Project.toml:40]: Need to add Compat here and in the test dependencies for `contains()`.

**Review by richardreeve — CHANGES_REQUESTED** [review]
Tiny edits - looks great to me though.

File: test/test_metrics.jl (hunk 1 above)
Comment (claireh93) [reply test/test_metrics.jl:6]: I did try this already, but it didn't work for me for some reason - see 2db1355

Comment (richardreeve) [reply test/test_metrics.jl:6]: Okay, I give up - I've tried really hard to get it to work, but there's something very weird going on I agree. However, I'm really not keen on this new definition of contains, so can we just change `contains(x, y)` to `occursin(y, x)` in the two places it's used...

File: test/test_metrics.jl [hunk 4]
```diff
@@ -3,6 +3,10 @@ module TestMetrics
...
+    contains(x, y) = occursin(y, x)
+end
+
```
Comment (richardreeve) [inline test/test_metrics.jl:9]: So delete this completely

**Review by richardreeve — CHANGES_REQUESTED** [review]
Sorry for being fussy - hopefully this replacement of contains with occursin will fix the problem.

**Review by richardreeve — APPROVED** [review]

**Pull request merged**